### PR TITLE
`incident_schedule` example Terraform

### PR DESCRIPTION
**Pull request by rorymalcolm**
- Provide an example of how best to use the incident.io schedule resource

### CODE REVIEW

File: examples/resources/incident_schedule/resource.tf [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+data "incident_user" "roryb" {
+  email = "rory@incident.io"
+}
+
+resource "incident_schedule" "testing_my_provider" {
+  name = "Testing Terraform"
+
+  // from tz database
```
Comment (lawrencejones) [inline examples/resources/incident_schedule/resource.tf:16]: Shall we expand this? Presumably this is an ISO format we can reference?

File: examples/resources/incident_schedule/resource.tf [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+data "incident_user" "martha" {
+  email = "martha@incident.io"
+}
+
+data "incident_user" "roryb" {
+  email = "rory@incident.io"
+}
+
+resource "incident_schedule" "testing_my_provider" {
```
Comment (lawrencejones) [inline examples/resources/incident_schedule/resource.tf:13]:
```suggestion
resource "incident_schedule" "primary_on_call" {
```

File: examples/resources/incident_schedule/resource.tf [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  name = "Testing Terraform"
```
Comment (lawrencejones) [inline examples/resources/incident_schedule/resource.tf:14]:
```suggestion
  name = "Primary On-call"
```

File: examples/resources/incident_schedule/resource.tf [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+  // from tz database
+  timezone = "Europe/London"
+
+  rotations = [{
+    // A string ID for a schedule rotation
+    id                = "01HPFH8T92MPGSQS5C1SPAF4V0"
```
Comment (lawrencejones) [inline examples/resources/incident_schedule/resource.tf:21]: How do I pick this value? Presumably we're not asking people to ULID generate it, right?

Is it not the case that most people will end up calling this something they create by hand like:
```
id = "primary"
name = "Primary"
```

File: examples/resources/incident_schedule/resource.tf (hunk 1 above)
Comment (rorymalcolm) [reply examples/resources/incident_schedule/resource.tf:16]: There's no ISO format afaik - `tz` is the scientific term! https://en.wikipedia.org/wiki/List_of_tz_database_time_zones

**Review by lawrencejones — APPROVED** [review]

**Pull request merged**